Makefile.am

```diff
@@ -305,6 +305,11 @@ bench_pmtud_SOURCES = \
 bench_pmtud_CFLAGS = $(libpacketgraph_dev_la_CFLAGS)
 bench_pmtud_LDFLAGS = libpacketgraph-dev.la
 
+bench_ip_fragment_SOURCES = \
+	tests/ip-fragment/bench.c
+bench_ip_fragment_CFLAGS = $(libpacketgraph_dev_la_CFLAGS)
+bench_ip_fragment_LDFLAGS = libpacketgraph-dev.la
```

Comment:
You should also add:
`EXTRA_bench_ip_fragment_DEPENDENCIES = libpacketgraph-dev.la`